src/main/java/com/conveyal/r5/profile/McRaptorSuboptimalPathProfileRouter.java

```diff
@@ -235,6 +245,10 @@ private boolean doOneRound () {
 
                         for (TripSchedule tripSchedule : pattern.tripSchedules) {
                             currentTrip++;
+                            //Skips trips which don't run on wanted date
```

Comment:
I'm not sure here if currentTrip needs to be incremented even if it is skipped. @mattwigway ?